src/main/java/mate/academy/springboot/swagger/dto/mapper/ProductMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.springboot.swagger.dto.mapper;
+
+import mate.academy.springboot.swagger.dto.ProductRequestDto;
+import mate.academy.springboot.swagger.dto.ProductResponseDto;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.stereotype.Component;
+
+@Component

```

Comment:
I think you forgot to create interface DtoMapper with generics. If our project will extend and there will be others enteties.